App/ProtoControllers/HautResponsable/ClotureExercice.php

```diff
@@ -43,7 +43,7 @@ public static function traitementClotureEmploye($employes, $typeConges, &$error,
     private static function updateSoldesEmploye($employe, $typeConges, $comment, \includes\SQL $sql, \App\Libraries\Configuration $config)
     {
         $return = true;
-        $soldesEmploye = \App\ProtoControllers\Utilisateur::getSoldesEmploye($sql, $config, $employe);
+        $soldesEmploye = \App\ProtoControllers\Utilisateur::getSoldesEmploye($sql, $employe);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\App\ProtoControllers\Utilisateur' in method 'updateSoldesEmploye'.](https://app.codacy.com/manual/Libertempo/web/pullRequest?prid=5122180)